scipy/spatial/transform/rotation.py

```diff
@@ -211,7 +211,9 @@ class Rotation(object):
     apply
     __mul__
     inv
+    identity
```

Comment:
Perhaps it's better to put factory methods near each other.

Reply:
Yep.  Fixed.